examples/workflow/pygrb/run_pygrb.sh

```diff
@@ -1,20 +1,25 @@
 #!/bin/sh
```

Comment:
This works with current example config files and current pygrb_make_offline_workflow executable